sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomModelInfo.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace Azure.AI.FormRecognizer.Custom
 {
-    public class CustomModelTrainingStatus
+    public class CustomModelInfo
```

Comment:
This type looks very similar to the generated type except the model id. You'll be able to get rid of `ConvertModelList` method if you use `[CodeGenSchema]` here.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/10390